build-scripts/gulp/translations.js

```diff
@@ -120,17 +130,14 @@ function lokaliseTransform(data, original, file) {
   return output;
 }
 
-gulp.task("clean-translations", async () => (await del).deleteSync([workDir]));
+gulp.task("clean-translations", async () => deleteSync([workDir]));
```

Comment:
These no longer need to be async?
```suggestion
gulp.task("clean-translations",() => deleteSync([workDir]));
```

Reply:
Without a callback, Gulp still needs a promise or stream to be returned though, right?

Reply:
yes, would be:
```js
(done) => {deleteSync([workDir]); done();}
```

Reply:
And one day convert our `gulp.task` to the new exports way 🙃

Reply:
Agree.  Would make jumping around in code a lot easier.  Will be a pain though because they all need to be renamed without hyphens.